src/locales/en/common.json

```
@@ -689,6 +689,7 @@
     }
   },
   "reset": {
+    "title": "Uninstall then reinstall ledger live",
```

```suggestion
    "title": "Uninstall then reinstall Ledger Live",
```
I would assume this is what we expect, we never lowercase ledger

---

The whole text is used in uppercase but yes we never know, thanks.